packages/openapi3/test/3.0CoverageData.json

```diff
@@ -0,0 +1,677 @@
+[
+  {
+    "scenariosMetadata": {
+      "commit": "71dc4ec41301d2b69a1546972f0aac1f24b5f93d",
+      "version": "0.1.0-alpha.16",
+      "packageName": "@typespec/http-specs"
+    },
+    "results": {
+      "Routes_InInterface": "pass",
+      "Routes_fixed": "pass",
+      "Routes_PathParameters_templateOnly": "pass",
+      "Routes_PathParameters_explicit": "pass",
+      "Routes_PathParameters_annotationOnly": "pass",
+      "Routes_PathParameters_ReservedExpansion_template": "not supported",
+      "Routes_PathParameters_ReservedExpansion_annotation": "not supported",
+      "Routes_PathParameters_SimpleExpansion_Standard_primitive": "pass",
+      "Routes_PathParameters_SimpleExpansion_Standard_array": "not supported",
+      "Routes_PathParameters_SimpleExpansion_Standard_record": "not supported",
+      "Routes_PathParameters_SimpleExpansion_Explode_primitive": "pass",
```

Comment:
What I was envisioning is that you still run the emitter on the specs to check it doesn't crash unexpectedly.

I think the flow should be as follow:
1. For each spec run the emitter
2. If the compile pass then its all good, count all scenarios in the file as a success(I don't think we need to explicitly list them)
3. If the compile fail then it looks up to the failing scenario mapping and report the manual reporting for those that either they are all failing or only part of them
4. Generate the final file.